### DISCUSSION TITLE
Permission requeset for image in documentation

### DISCUSSION THREAD

**Discussion by ericbrow** [TPC-Council/HammerDB#89] [Q&A] [answered]
First I'd like to thank the HammerDB team for their great work.
I'm working on my doctoral dissertation comparing the performance of MySQL on virtual machines on the three major cloud providers.  In reading the academic literature, the TPC benchmarks are most frequently used for measuring throughput.   Searching around, of course your great tool seems to be the best fit.  In explaining how I will be gathering data in my dissertation, I decided the image 3.2 in section 3.5 best describes what's going on under the hood ([found here](https://www.hammerdb.com/docs/ch03s05.html)).
Could I get approval for using this image in my dissertation?  I'd be happy to provide any other information or documentation.
Thank you for your attention to this request.

**Comment by sm-shaw** [comment, marked as answer]
Hi, yes this image is fine to reuse.
